usaspending_api/disaster/management/sql/disaster_covid19_file_f_grants.sql

```diff
@@ -1,108 +1,107 @@
-    "broker_subaward"."sub_high_comp_officer2_full_na" AS "subawardee_highly_compensated_officer_2_name",
-    "broker_subaward"."sub_high_comp_officer2_amount" AS "subawardee_highly_compensated_officer_2_amount",
-    "broker_subaward"."sub_high_comp_officer3_full_na" AS "subawardee_highly_compensated_officer_3_name",
-    "broker_subaward"."sub_high_comp_officer3_amount" AS "subawardee_highly_compensated_officer_3_amount",
-    "broker_subaward"."sub_high_comp_officer4_full_na" AS "subawardee_highly_compensated_officer_4_name",
-    "broker_subaward"."sub_high_comp_officer4_amount" AS "subawardee_highly_compensated_officer_4_amount",
-    "broker_subaward"."sub_high_comp_officer5_full_na" AS "subawardee_highly_compensated_officer_5_name",
-    "broker_subaward"."sub_high_comp_officer5_amount" AS "subawardee_highly_compensated_officer_5_amount",
+    "legal_entity_state_code" AS "prime_awardee_state_code",
+    "legal_entity_state_name" AS "prime_awardee_state_name",
+    "legal_entity_zip" AS "prime_awardee_zip_code",
+    "legal_entity_congressional" AS "prime_awardee_congressional_district",
+    "legal_entity_foreign_posta" AS "prime_awardee_foreign_postal_code",
+    "business_types" AS "prime_awardee_business_types",
+    "place_of_perform_scope" AS "prime_award_primary_place_of_performance_scope",
+    "place_of_perform_city_name" AS "prime_award_primary_place_of_performance_city_name",
+    "place_of_perform_state_code" AS "prime_award_primary_place_of_performance_state_code",
+    "place_of_perform_state_name" AS "prime_award_primary_place_of_performance_state_name",
+    "place_of_performance_zip" AS "prime_award_primary_place_of_performance_address_zip_code",
+    "place_of_perform_congressio" AS "prime_award_primary_place_of_performance_congressional_district",
+    "place_of_perform_country_co" AS "prime_award_primary_place_of_performance_country_code",
+    "place_of_perform_country_na" AS "prime_award_primary_place_of_performance_country_name",
+    "award_description" AS "prime_award_base_transaction_description",
+    (SELECT STRING_AGG(DISTINCT CONCAT(U0."cfda_number", ': ', U0."cfda_title"), '; '  ORDER BY  CONCAT(U0."cfda_number", ': ', U0."cfda_title")) AS "total" FROM "transaction_fabs" U0 INNER JOIN "transaction_normalized" U1 ON (U0."transaction_id" = U1."id") WHERE U1."award_id" = ("subaward_search"."award_id") GROUP BY U1."award_id") AS "prime_award_cfda_numbers_and_titles",
+    "subaward_type" AS "subaward_type",
+    "internal_id" AS "subaward_fsrs_report_id",
+    "subaward_report_year" AS "subaward_fsrs_report_year",
+    "subaward_report_month" AS "subaward_fsrs_report_month",
+    "subaward_number" AS "subaward_number",
+    "subaward_amount" AS "subaward_amount",
+    "sub_action_date" AS "subaward_action_date",
+    EXTRACT(YEAR FROM ("action_date") + INTERVAL '3 months') AS "subaward_action_date_fiscal_year",
```

Comment:
In the other covid SQL for contracts, this was
```
EXTRACT (YEAR FROM ("subaward_search"."sub_action_date")
```

Should it be similarly retargeting that new field here too?